### PR TITLE
Fixed trustchain crawler lambda

### LINKED ISSUE #638: [asyncio] Typo in variable name
In [this](https://github.com/Tribler/py-ipv8/blob/babcdae067de773526ef25eb82eef76b35268f37/scripts/trustchain_crawler_plugin.py#L157) the lambda holds a parameter named `sign`, yet it uses `sig` inside the lambda.
```python
signal.signal(signal.SIGTERM, lambda sign, _: ensure_future(signal_handler(sig)))
```
Also the line right above it has `sig` as its parameter name instead of `sign`. In addition `sig` is used in the rest of the function. Is it a typo?

Running flake8 detects `sig` in the line as undefined.

### PR DESCRIPTION
**Pull request by Solomon1732**
Fixes #638 .

### CODE REVIEW

**Comment by devos50** [comment]
ok to test

**Comment by Solomon1732** [comment]
Looking here I can't seem to see which unittests failed on linux. It doesn't reveal anything.
https://jenkins-ci.tribler.org/job/ipv8/job/test_ipv8_PR_unittests_linux/1497/

**Comment by Solomon1732** [comment]
Hmm. Seems like the failing test isn't related to the PR. Can someone retest this please?

The failing test is this:
https://github.com/Tribler/py-ipv8/blob/asyncio/ipv8/test/attestation/wallet/irmaexact/test_builder.py#L72
https://github.com/Solomon1732/py-ipv8/blob/f_rename_variable/ipv8/test/attestation/wallet/irmaexact/test_builder.py#L72

**Comment by devos50** [comment]
retest this please

**Comment by devos50** [comment]
This is indeed not related to your PR. I will make a new issue for the observed bug 👍

File: scripts/trustchain_crawler_plugin.py [hunk 1]
```diff
@@ -6,17 +6,14 @@
...
+from asyncio import all_tasks, ensure_future, gather, get_event_loop, sleep
 
 from ipv8.REST.rest_manager import RESTManager
 from ipv8.attestation.trustchain.settings import TrustChainSettings
 
-import yappi
-
 from ipv8_service import IPv8
 
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
```
Comment (qstokkink) [inline scripts/trustchain_crawler_plugin.py]: This is wrong.

The path modification to allow imports from `ipv8.*` should be before the imports from `ipv8.*`.

Comment (Solomon1732) [reply scripts/trustchain_crawler_plugin.py]: I changed the order because I saw it was a flake8 violation. I will change it back, however 👍

**Comment by Solomon1732** [comment]
No problem 👍

**Comment by qstokkink** [comment]
retest this please

**Comment by Solomon1732** [comment]
I see the test that failed is this https://jenkins-ci.tribler.org/job/ipv8/job/test_ipv8_PR_unittests_windows/1491/console
It's related to #641

**Comment by qstokkink** [comment]
Hmm.. I see someone switched the Windows tester to Python 3 without informing me. It's probably a Python 3 specific bug then, I'll reopen #641.

**Pull request merged**